packages/kit/src/exports/public.d.ts

```diff
@@ -1,7 +1,6 @@
-/// <reference types="svelte" />
-/// <reference types="vite/client" />
-
-import './ambient.js';
+import 'svelte';
+import 'vite/client';
```

Comment:
are there specific types we can import from these? I'm not that familiar with typescript, but as a neophyte that seems like it could be nicer than having to import everything

Reply:
We're importing them ambiently so that we pick up `declare module "*.svelte"` and `declare module "*.jpg"` etc

Reply:
(This is no different to what currently happens, except we're using `import` rather than `reference`, because `dts-buddy` doesn't understand `reference` yet and AFAICT they're equivalent)

Reply:
I wonder if it's worth a comment explaining what we get from them?
```suggestion
import 'svelte';      // pick up `declare module "*.svelte"`
import 'vite/client'; // pick up `declare module "*.jpg"`, etc.
```